src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -171,7 +171,7 @@ protected override int GetPositionBeforeParameterListClosingBrace(SyntaxNode mat
             {
                 ParameterListSyntax parameterListSyntax => parameterListSyntax.CloseParenToken.SpanStart,
                 BracketedParameterListSyntax bracketedParameterListSyntax => bracketedParameterListSyntax.CloseBracketToken.SpanStart,
-                _ => throw new ArgumentException("Unexpected SyntaxNode", nameof(matchingNode))
+                _ => matchingNode.SpanStart // e.g. unparenthesized lambda parameters
```

Comment:
This whole method can probably be gotten rid of, since it was more necessary for the completion list aspect which we pulled out. But this is a minimal fix for the crash.